src/main/java/net/sf/jabref/gui/entryeditor/EntryEditor.java

```diff
@@ -330,6 +340,41 @@ private void addGeneralTabs() {
         }
     }
 
+    private void addSpecialTabs() {
+
+        // MathSciNet Review
+        if(entry.hasField("MRNumber")) {
+            String mrNumberRaw = entry.getField("MRNumber");
+            // Take everything before whitespace or open bracket, so something like `619693 (82j:58046)` gets parsed correctly
+            String mrNumber = StringUtil.tokenizeToList(mrNumberRaw, " (").get(0);
+
+            JFXPanel reviewPane = new JFXPanel();
+            tabbed.addTab(Localization.lang("MathSciNet Review"), reviewPane);
+            tabs.add(reviewPane);
+
+            // Execute on JavaFX Application Thread
+            Platform.runLater(() -> {
+                StackPane root = new StackPane();
```

Comment:
I now switched to the ProgressIndicator (instead of bar) and already hide it at 70% completion.